apps/condo/domains/miniapp/schema/SendAppPushMessageService.test.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Generated by `createservice miniapp.SendAppPushMessageService --type mutations`
+ */
+
+const { makeLoggedInAdminClient, makeClient } = require('@open-condo/keystone/test.utils')
+const { expectToThrowAccessDeniedErrorToObj, expectToThrowAuthenticationErrorToObjects } = require('@open-condo/keystone/test.utils')
+
+const { sendAppPushMessageByTestClient } = require('@condo/domains/miniapp/utils/testSchema')
+ 
+//TODO(KEKMUS) write normal tests when I add all errors to sendAppPushMessageService
+describe('SendAppPushMessageService', () => {
```

Comment:
You also need cases:

 - management company admin user trying to send a message
 - some staff user trying to send a message
 - staff user trying to send message to himself
 - resident user trying to send message to himseld